### PR TITLE
evidence: replace mock evidence with mocked duplicate vote evidence

### PR DESCRIPTION
**Pull request by cmwaters**
Branches of #5003 so will wait for this to be merged first

Closes: #5024

### CODE REVIEW

File: evidence/pool_test.go [hunk 1]
```diff
@@ -11,7 +11,7 @@ import (
 
 	dbm "github.com/tendermint/tm-db"
 
-	"github.com/tendermint/tendermint/crypto"
+	// "github.com/tendermint/tendermint/crypto"
```
Comment (marbar3778) [inline evidence/pool_test.go:14]:
```suggestion
```

File: evidence/pool.go [hunk 2]
```diff
@@ -216,6 +216,7 @@ func (evpool *Pool) AddEvidence(evidence types.Evidence) error {
 			return fmt.Errorf("failed to verify %v: %w", ev, err)
 		}
 
+		evpool.logger.Info("Hi")
```
Comment (marbar3778) [inline evidence/pool.go:219]:
```suggestion
```

File: consensus/reactor_test.go [hunk 3]
```diff
@@ -669,7 +667,7 @@ func timeoutWaitGroup(t *testing.T, n int, f func(int), css []*State) {
 
 	// we're running many nodes in-process, possibly in in a virtual machine,
 	// and spewing debug messages - making a block could take a while,
-	timeout := time.Second * 300
+	timeout := time.Second * 60
```
Comment (melekes) [inline consensus/reactor_test.go:670]: what's the reason for changing this?

Comment (melekes) [reply consensus/reactor_test.go:670]: I thought we've increased it because the tests were failing, no?

Comment (cmwaters) [reply consensus/reactor_test.go]: sorry yes I reverted it

**Pull request merged**